### ISSUE TITLE
Implement random search for hyperparameter optimization

### ISSUE THREAD

**Issue by tommartensen** [tommartensen/fake-news-detector#13]
This should be done using the validation set.
